bots/images/scripts/fedora-24.setup

```diff
@@ -41,7 +41,6 @@ storaged-lvm2 \
 storaged-iscsi \
 subscription-manager \
 tuned \
-virt-install \
```

Comment:
I'd really keep virt-install on our test images and in the Vagrantfile. First, this PR doesn't rebuild all the affected images for that (and I don't think we should block on that either, given how hard it is to land it), and second I'm rather sure we are going to use virt-install for machine creation in Cockpit and then we'll need it again.

At that point we should also move the tests back to using virt-install instead of raw XML, but we can see how image creation pans out.